requirements.txt

```diff
@@ -8,7 +8,7 @@ requests>=2.12.0,==2.*
 jsonschema>=2.5.1,<=3
 functools32>=3.2,<=3.99 ; python_version == '2.7'
 six>=1.10,<=1.99
-joblib>=0.11,<=0.13
+joblib>=0.11,<0.15
 pubnub>=4.0,<=4.99
 cloudpickle>=0.2,<=1 ; python_version != '3.4'
```

Comment:
Just tested `pip install cloudpickle>=0.2,<=1` in a fresh virtual environment (Python 3.7, pip v19.3.1), and I got cloudpickle v1.0.0, which suggests the `<=1` seems like a bug if we meant to allow cloudpickle v1.2.2. How about `cloudpickle>=0.2,<2`? (I tested `pip install cloudpickle>=0.2,<2` and got cloudpickle v1.2.2 as desired.)

Also, apart from `requirements.txt` here, it's important (perhaps more so, for production purposes) to make the changes in `setup.py` as well.

Reply:
Yeah, I changed the wrong line :facepalm:. Good point about setup.py as well.